python/replicate/experiment.py

```diff
@@ -55,6 +61,7 @@ class Experiment:
     host: str
     command: str
     config: dict
+    status: str
```

Comment:
Do we need a `status` field on `Experiment`? It might be bug-prone to try to keep this field consistent. Instead, could we use the `is_running()` method to retrieve the status on demand?

Reply:
I added the field so that the `list()` method would return a list of experiments with the `status` field just like a `replicate ls` would. But I think using the method `is_running()` to retrieve the status on demand is better as you mentioned. Will make the needful changes.